DoctorWho.Db/Models/Episode.cs

```diff
@@ -1,12 +1,39 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
 
 namespace DoctorWho.Db.Models
 {
-    internal class Episode
+    public class Episode
     {
+        public Episode()
+        {
+            EpisodeType=string.Empty;
+            Title = string.Empty;
+            Notes= string.Empty;
+            Enemies=new List<Enemy>();
+            Companions=new List<Companion>();   
+            
+        }
+        [Required]
```

Comment:
> By convention, a property named Id or <type name>Id will be configured as the primary key of an entity.

[Configuring a primary key - EF Core documentation](https://learn.microsoft.com/en-us/ef/core/modeling/keys?tabs=data-annotations#configuring-a-primary-key)

I'd agree with adding the `[Key]` attribute explicitly, but you don't need a `[Required]` attribute. A key is required by default.